daemons/pacemakerd/pcmkd_subdaemons.c

```diff
@@ -82,6 +82,7 @@ static char *opts_vgrind[] = { NULL, NULL, NULL, NULL, NULL };
 
 crm_trigger_t *shutdown_trigger = NULL;
 crm_trigger_t *startup_trigger = NULL;
+time_t subdaemon_check_progress;
```

Comment:
I like to explicitly initialize globals for readability

Reply:
Something inside me had refused to initialized to 0 ;-)
But lacking a nice invalid-value initializer - at least to my knowledge - I'll go that route.